README.md

```diff
@@ -10,3 +10,16 @@ All `obal` actions should also work with plain Ansible when called like `ansible
 
 - `python` (2 or 3)
 - `ansible`
+
+## Using Obal via Container
+
+Obal and all it's required packages are available in a container that can be used locally or in build environments. The users Koji credentials and configuration must be mounted into the container alongside of mounting the packaging project into `/opt/packaging` to work. Note the examples below assume SELinux is disabled.
+
+With `podman`:
+
+    podman run -v `pwd`:/opt/packaging -v ~/.koji:/root/.koji obal:latest scratch katello
+
+With `docker`:
+
+		docker run -v `pwd`:/opt/packaging -v ~/.koji:/root/.koji obal:latest scratch katello
```

Comment:
The indentation is off, but I wonder if we really need two identical examples with a different binary name? I mean, `podman` is designed to be a drop-in replacement for `docker`, right?